CONTRIBUTING.md

```diff
@@ -80,20 +88,55 @@ NB: If you want to change your Meilisearch credentials or the plugins options yo
 Each PR should pass the tests and the linter to be accepted.
 
 ```bash
+curl -L https://install.meilisearch.com | sh # download Meilisearch
+./meilisearch --master-key=masterKey --no-analytics=true # run Meilisearch
+
 # Tests
-npm run test
+yarn test
 
 # Tests in watch mode
-npm run test:watch
+yarn test:watch
 
 # Linter
-npm run lint
+yarn lint
 
 # Linter with fixing
-npm run lint:fix
+yarn lint:fix
 
 # Build the project
-npm run build
+yarn build
+```
+
+### Run the backfilled-data script
+
+Run the import script using [`npx` (the Node Package Runner)](https://www.npmjs.com/package/npx) via `yarn` (the Node Package Manager).
```

Comment:
Not sure I understand this sentence. What do you mean? Npx and yarn are two separate things

Reply:
Yes, but you will need `yarn` or `npm` to run and install `npx`

Reply:
See beneeth, you don't need to specify how you installed npx, The link you provided is enough. Also npx is downloaded via npm not yarn! But so is a lot of things and they may even be other ways to download npx! 
```suggestion
Run the import script using [`npx` (the Node Package Runner)](https://www.npmjs.com/package/npx).
```